docker-compose.yml

```diff
@@ -31,6 +36,19 @@ services:
     ports:
       - "127.0.0.1:6379:6379"
 
+  metricity:
+    << : *logging
+    << : *restart_policy
+    depends_on:
+      postgres:
+        condition: service_healthy
+    image: ghcr.io/python-discord/metricity:latest
+    env_file:
+      - .env
+    environment:
+      DATABASE_URI: postgres://pysite:pysite@postgres:5432/pysite
+      skip_metricity: true  # Delete this to run the metricity bot too
```

Comment:
Is there some way of accepting this from .env and defaulting to true or will docker always use the val from here?

If it stays I think it'd be worth mentioning the override file here as changing the value here is not a permanent solution as it's tracked

Reply:
I think this is possible now that I've pushed https://github.com/python-discord/metricity/pull/7/commits/9a5d4ce294c3d0a9d588bc4e1435606690e8c02f will test it tonight

Reply:
https://github.com/python-discord/bot/pull/1820/commits/4d7496d20c867b1d5201e7d60d3c22d18e2a21a3 Much nicer interface, thanks